README.md

```diff
@@ -82,13 +75,13 @@ ansible-galaxy collection build --force
 ansible-galaxy collection install cisco-aci-* --force
 ```
 
-### Second Approach
+#### Second Approach
 
 Go to [ansible-aci Actions](https://github.com/CiscoDevNet/ansible-aci/actions)
```

Comment:
Should we remove this part from the docs, since actions have not been running stable.

Reply:
Perhaps we can push a nightly build on the ansible galaxy page: https://galaxy.ansible.com/ui/repo/published/cisco/aci/

Reply:
I agree with removing this part. The builds do produce a collection to download but due to the current integration tests they are always red.
Not sure anyone would be comfortable using those.
Pushing a nightly build can be a later task.

Reply:
Removed this part